svgnative/CMakeLists.txt

```diff
@@ -234,17 +234,17 @@ file(GLOB gl_headers ${PROJECT_SOURCE_DIR}/include/*.h)
 set(text_port)
 if(USE_TEXT)
 file(GLOB text_port
-    ports/string/StringSVGRenderer.h
-    ports/string/StringSVGRenderer.cpp
+    src/ports/string/StringSVGRenderer.h
+    src/ports/string/StringSVGRenderer.cpp
```

Comment:
Leaving the ports separate of the actual source was a requirement by other projects. I agree that it is odd, but I need to check if these requirements are still relevant. Could you leave the ports/ folder in root for now?